### DISCUSSION TITLE
report file meaning

### DISCUSSION THREAD

**Discussion by DiracZhu1998** [broadinstitute/Drop-seq#276] [Q&A] [answered]
Dear Drop-seq developer, 
Thank you for giving us such wonderful tools. 
Would you please explain "polyA_trimming_report.txt" and "adapter_trimming_report.txt" report file for me ? I couldn't figure it by myself.
[polyA_trimming_report.txt](https://github.com/broadinstitute/Drop-seq/files/7414636/polyA_trimming_report.txt)
[adapter_trimming_report.txt](https://github.com/broadinstitute/Drop-seq/files/7414637/adapter_trimming_report.txt)
Thanks a lot!

**Comment by alecw** [comment, marked as answer]
adapter_trimming_report.txt is produced by TrimStartingSequence.  polyA_trimming_report.txt is produced by PolyATrimmer.  

Both reports are histograms in which the BIN column represents the length of the trimmed sequence, and VALUE represents the number of reads that were trimmed that amount.

**Comment by DiracZhu1998** [comment]
I get it! Thank you !
